apps/pair-matching/src/components/PairMatching.scss

```diff
@@ -0,0 +1,181 @@
+  src: url('../assets/fonts/IRANYekanWebMedium.woff2');
+}
+
+.pair-matching * {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  direction: rtl;
+  --mdc-theme-primary: #651fff;
+  --mdc-typography-font-family: IranYekan;
+  --mdc-typography-letter-spacing: 0;
+}
+
+mwc-button{
+  --mdc-typography-button-letter-spacing: 0;
+}
+
+.wrapper {
+  padding: 1rem 30%;
+}
+
+.pair-matching * {
+  font-family: IranYekan, sans-serif;
+}
+
+.pair-matching .game {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: 1%;
+  position: relative;
+}
+
+.pair-matching .game img {
+  max-width: 100%;
+}
+
+.pair-matching .game .card {
+  position: relative;
+  transform-style: preserve-3d;
+  transition: transform .3s ease-in-out;
+  border-radius: 4px;
+  box-shadow: 0 0 .5rem $whitesmoke;
+  margin-bottom: 1%;
+  cursor: pointer;
+}
+
+.pair-matching .game .card .front, .pair-matching .game .card .back {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  left: 0;
+  top: 0;
+  border-radius: 4px;
+}
+
+.pair-matching .game .card .front {
+  background: $whitesmoke;
+  z-index: 2;
+}
+
+.pair-matching .game .card .back {
+  background: #fefefe;
+  transform: rotateY(180deg);
+  padding: 1rem;
+}
+
+.pair-matching .game .card.flipped {
+  transform: rotateY(180deg);
+}
+
+.pair-matching .game .card.solved {
+  transform: rotateY(180deg);
+  opacity: .3;
+}
+
+.pair-matching .game .card.solved .front {
+  z-index: -1;
+}
+.pair-matching .game .card.solved .front svg{
+  width: 100%;
+}
+
+.pair-matching .game.size4 .card {
+  width: 24.2%;
+  padding-top: 24.2%;
+}
+
+.pair-matching .game.size5 .card {
+  width: 19.2%;
+  padding-top: 19.2%;
+}
+
+.pair-matching .game.size6 .card {
+  width: 15.8%;
+  padding-top: 15.8%;
+}
+.pair-matching .game.size6 .card .back{
+  padding: .5em;
+}
+
+.pair-matching .settings {
+  display: flex;
+  margin-bottom: 1rem;
+}
+
+.pair-matching .settings mwc-select {
+  direction: ltr;
+}
+
+.pair-matching .time {
+  direction: ltr;
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  background: #f5f5f5;
+  padding: 3px 15px;
+  border-radius: 4px 4px 0 0;
+  font-size: 1rem;
+  border-bottom: 1px solid #777;
+}
+
+.pair-matching .time div {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  flex: 0 50%;
+}
+
+.pair-matching .game .overlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background: rgba(255, 255, 255, .5);
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+  align-items: center;
+  gap: 1rem;
+  padding-top: 40%;
+}
+
+.pair-matching .game .overlay span {
+  font-size: 1.2rem;
+}
+
+@media screen and (max-width: 1024px) {
+  .wrapper{
+    padding: 1rem 10%;
+  }
+}
+
+@media screen and (max-width: 768px) {
+  .wrapper{
+    padding: 1rem;
+  }
+  .pair-matching .settings{
+    flex-direction: column;
+  }
+  .pair-matching .game .card .back{
+    padding: .2rem;
+  }
+}
```

Comment:
I suggest adding a free line to the end of the file.

Reply:
done